src/loom.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Get the loom id.
+ * @param {string} urlString - the url from which you want to extract the id
+ * @returns {string|undefined}
+ */
+export default function loom(urlString) {
+	// Parse basic url and embeds
+	const basicReg
+		= /(https?:\/\/)?(www\.)?loom\.com\/?(.*\/)?([\d)([a-z]{32})\??.*/;
```

Comment:
Seems like this RegEx is way to strict which might present problems (false positives, not flexible, etc), so I think you can really simplify this RegEx to make it focus on the specific strings you are looking for, and don't force length limits on the hashed id. something like (this is not tested, so please test it or modify it accordingly):
```suggestion
		= /^https?:\/\/(?:www\.)?loom\.com\/(?:share|embed)\/([\da-zA-Z]+);
```